contents/blog/bad-behaviors.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Non-obvious behaviors that will kill your startup
+date: 2024-10-10
+author:
+  - andy-vandervell
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/353483136_5f00599e_346c_44d4_a94c_2cd85c9031a0_3c65d69b45.jpg
+featuredImageType: full
+tags:
+  - Culture
+  - People
+crosspost:
+  - Newsletter
+  - Founders
+---
+
+## 1. Not praising people for day-to-day work
+
+![star](https://res.cloudinary.com/dmukukwp6/image/upload/star_hog_0c334584ca.png)
```

Comment:
This was fine for the newsletter but now is two massive images in a row for the blog which is weird.

![CleanShot 2024-10-10 at 12 39 03](https://github.com/user-attachments/assets/57c7bfd2-1365-4dbe-a58a-5fd22daaea40)